data/data/alibabacloud/cluster/vpc/outputs.tf

```diff
@@ -22,12 +22,16 @@ output "eip_ip" {
   value = alicloud_eip_address.eip.ip_address
 }
 
-output "slb_ids" {
-  value = [alicloud_slb_load_balancer.slb_external.id, alicloud_slb_load_balancer.slb_internal.id]
```

Comment:
I think we can keep the same `slb_ids` output.
```
output "slb_ids" {
  value = concat(alicloud_slb_load_balancer.slb_external[*].id, [alicloud_slb_load_balancer.slb_internal.id])
}
```

Reply:
Have updated

Reply:
@jianli-wei Maybe need your help to test the updated code